routers/web/explore/repo.go

```diff
@@ -129,10 +129,17 @@ func RenderRepoSearch(ctx *context.Context, opts *RepoSearchOptions) {
 		return
 	}
 
+	languages, err := repo_model.GetPrimaryRepoLanguageList(ctx)
+	if err != nil {
+		ctx.ServerError("SearchRepository", err)
+		return
+	}
+
 	ctx.Data["Keyword"] = keyword
 	ctx.Data["Total"] = count
 	ctx.Data["Repos"] = repos
 	ctx.Data["IsRepoIndexerEnabled"] = setting.Indexer.RepoIndexerEnabled
+	ctx.Data["Languages"] = languages
```

Comment:
How about renaming this as `ProgramLanguages` to reduce the possibility of conflicting with UI Languages.